src/styles/header.scss

```diff
@@ -0,0 +1,73 @@
+:root {
+  --linkcolor: #00acdc;
+}
```

Comment:
Do you need this color with root selector?
If you have the separate file with variables